### ISSUE TITLE
`range` and `loc` options are always `true`

### ISSUE THREAD

**Issue by ehmicky** [typescript-eslint/typescript-eslint#694]
**What code were you trying to parse?**

```ts
typescriptEstree.parse('true', {range: false, loc: false})
```

**What did you expect to happen?**

The returned node should not include the `range` or `loc` properties.

**What actually happened?**

```json
{
  "type": "Program",
  "body": [
    {
      "type": "ExpressionStatement",
      "expression": {
        "type": "Literal",
        "value": true,
        "raw": "true",
        "range": [
          0,
          4
        ],
        "loc": {
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 1,
            "column": 4
          }
        }
      },
      "range": [
        0,
        4
      ],
      "loc": {
        "start": {
          "line": 1,
          "column": 0
        },
        "end": {
          "line": 1,
          "column": 4
        }
      }
    }
  ],
  "sourceType": "script",
  "range": [
    0,
    4
  ],
  "loc": {
    "start": {
      "line": 1,
      "column": 0
    },
    "end": {
      "line": 1,
      "column": 4
    }
  }
}
```

**Versions**

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/typescript-estree` | `1.11.0` |
| `TypeScript`                           | `3.5.3` |
| `node`                                 | `12.6.0` |
| `npm`                                  | `6.10.0` |

**Comment by M4rk9696** [comment]
Hi @bradzacher,
Can I pick this up? Had a look into the code, I thought of this approach can you comment on it.

I'll update the `ConverterOptions` to include the `range` and `loc` options.
https://github.com/typescript-eslint/typescript-eslint/blob/c2ad091c2a91349c0db1991925f5f3098b942d58/packages/typescript-estree/src/convert.ts#L25-L29

Will then pass the options to convertor instance here
https://github.com/typescript-eslint/typescript-eslint/blob/c2ad091c2a91349c0db1991925f5f3098b942d58/packages/typescript-estree/src/ast-converter.ts#L23-L27

Then using the options I exclude `range` and `loc` from getting added when converting nodes in convert.ts

**Comment by ehmicky** [comment]
Thanks for working on this! :)
